MapboxNavigation/RouteMapViewController.swift

```diff
@@ -459,7 +459,7 @@ func defaultFeedbackHandlers(source: FeedbackSource = .user) -> (send: FeedbackV
         endOfRoute.dismissHandler = { [weak self] (stars, comment) in
             guard let rating = self?.rating(for: stars) else { return }
             self?.routeController.setEndOfRoute(rating: rating, comment: comment)
-            self?.dismiss(animated: true, completion: nil)
+            self?.delegate?.mapViewControllerDidCancelNavigation(self!)
```

Comment:
Perhaps we should add a new `…didEndNavigation()`? /cc @JThramer

Reply:
@frederoni agreed, It's time. Although what about something like `didEndNavigation(cancelled: Bool)`?

Reply:
`self?` is evaluated before `self!` so this could arguably be considered a safe unwrap.